electron/src/ipc/_ipc.ts

```diff
@@ -1,4 +1,5 @@
 import { ipcMain } from "electron"
+import pick = require("lodash.pick")
```

Comment:
Is there a particular reason for this `import pick = require()` syntax? Should work without.

Reply:
No there is no reason, I guess it was added by my VSCode while auto-importing.